converter/src/main/java/gov/cms/qpp/conversion/util/MeasureConfigHelper.java

```diff
@@ -27,6 +27,22 @@ public static MeasureConfig getMeasureConfig(Node node) {
 		return MeasureConfigs.getConfigurationMap().get(measureId);
 	}
 
+	/**
+	 * Gets the electronic measure id by uuid or defaults to empty if none exists
+	 *
+	 * @param uuid identifier used to fined the electronic measure id
+	 * @return electronic measure id
+	 */
+	public static String getMeasureConfigIdByUuidOrDefault(String uuid) {
+		String electronicMeasureId = "";
```

Comment:
why return a blank? What if config.getElectronicMeasureId() returns null

Reply:
Just to initialize it `electronicMeasureId` with a fast filler. I changed it to null however.